### PR TITLE
Order Completed Events "fan-out"

### PR DESCRIPTION
**Pull request by edubkendo**
In order to make it easier to segment on individual product properties,
order completed events will iterate over the array of products, sending
one Ordered Product event for each product, as well as an Order Completed event.

**What does this PR do?**
Creates parity for web device connection mode with https://github.com/segmentio/analytics-cloud-integrations/pull/64

**Are there breaking changes in this PR?**
No, this PR should only create new events, not change anything else about how events are currently processed.

**Any background context you want to provide?**
https://github.com/segmentio/analytics-cloud-integrations/pull/61

**Is there parity with the server-side/android/iOS integration components (if applicable)?**
This should create parity with the cloud connection mode integration

**Does this require a new integration setting? If so, please explain how the new setting works**
No.

### CODE REVIEW

File: integrations/vero/lib/index.js [hunk 1]
```diff
@@ -108,6 +108,27 @@ Vero.prototype.track = function(track) {
...
+
+Vero.prototype.orderCompleted = function(track) {
+  var products = track.properties().products;
+  if (products && Array.isArray(products)) {
+    for (var x = 0; x < products.length; x++) {
+      push('track', 'Ordered Product', products[x], { source: 'segment' });
+    }
+  }
+
+  push('track', track.event(), track.properties(), { source: 'segment' });
```
Comment (mericsson) [inline integrations/vero/lib/index.js]: Does this also need to call 

```
  // check for tags and either add or remove.
  var tags = track.options('Vero').tags;	  var tags = track.options('Vero').tags;
  if (tags) this.addOrRemoveTags(tags);	  if (tags) this.addOrRemoveTags(tags);
```

Comment (edubkendo) [reply integrations/vero/lib/index.js]: Good catch, thanks!

**Comment by mericsson** [comment]
Closing in favor of https://github.com/segmentio/analytics.js-integrations/pull/454

**Comment by mericsson** [comment]
Thanks for contributing!

**Pull request closed without merging**